loadrunner_professional/assets/logs/loadrunner_professional.yaml

```diff
@@ -0,0 +1,71 @@
+id: loadrunner_professional
+metric_id: loadrunner_professional
+facets: null
+pipeline:
+  type: pipeline
+  name: LoadRunner Scenario Information Pipeline
+  enabled: true
+  filter:
+    query: "source:loadrunner "
+  processors:
+    - type: attribute-remapper
+      name: LoadRunner Hostname Tag Creation
+      enabled: true
+      sources:
+        - hostname
+      sourceType: attribute
+      target: lr_hostname
+      targetType: tag
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: LoadRunner Service Tag Creation
+      enabled: true
+      sources:
+        - service
+      sourceType: attribute
+      target: lr_service
+      targetType: tag
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: LoadRunner Version Tag Creation
+      enabled: true
+      sources:
+        - scenario_started.general.version
+        - scenario_ended.general.version
+      sourceType: attribute
+      target: lr_version
```

Comment:
Can you please remap to the standard datadog `version` tag ? See datadog standard attributes [doc](https://docs.datadoghq.com/logs/log_configuration/attributes_naming_convention/)